extensions/wikia/Email/Controller/OtherEmailConfirmationController.class.php

```diff
@@ -0,0 +1,160 @@
+<?php
+
+namespace Email\Controller;
+
+use Email\EmailController;
+use Email\Check;
+
+abstract class AbstractEmailConfirmationController extends EmailController {
```

Comment:
Why was the EmailConfirmationController split in 2 like this? Is there a reason we shouldn't try and keep it in one related file like it was before?

If there is, can we at least change the name of this file? `OtherEmailConfirmationController` isn't very descriptive